src/app/controllers/CustomerController.ts

```diff
@@ -47,19 +47,16 @@ export interface Customer
     LoyaltyFunction,
     CommonCustomerProperties {}
 
-export interface CustomerOptions {
-  profile_picture?: string;
-  birthday?: string;
+export interface CustomerOptions extends Pick<CommonCustomerProperties, 'profile_picture' | 'birthday'>{
```

Comment:
Where is the Partial? Aren't these optional?

Reply:
Ok, So can i use `Partial<Pick<CommonCustomerProperties, 'profile_picture' | 'birthday'`.

My understanding is Partial will return T or undefined? is that right?

Reply:
In the Customer interface is missing 3 keys according with GraphQL schema. ( supercharge_coupon, deliverer_profile and delivery_adresses)